src/cliActions.js

```diff
@@ -136,9 +137,19 @@ const streamDApp = ({ pw }, signingKeyFile, devMode) =>
         // read signing key
         const rawSigningKey = fs.readFileSync(signingKeyFile, 'utf8');
         const signingKey = JSON.parse(rawSigningKey);
-        watchAndStreamBundleData(devMode, signingKey, ({ content }) => {
-          console.log({ content, singingPrivateKey, signingKey }); // TODO: need to process result
-        });
+        watchAndStreamBundleData(
+          devMode,
+          { ...signingKey, singingPrivateKey },
+          ({ content }) => {
```

Comment:
`content` is not a good variable name. `content` can be everything

Reply:
done